pages/news/2020-XX-XX-new-in-2-3-cmake.html

```diff
@@ -25,14 +25,17 @@
   We managed to reduce the line count from approximately 8000 lines of Python code down to a 2300 line `CMakeLists.txt` file plus modules - all in all we reduces the line count by almost 3000 lines.
 - If you prefer a graphical "build" button over compiling Mixxx on the command line, this is now possible:
   IDEs such as [KDevelop](https://www.kdevelop.org/features), [Visual Studio](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=vs-2019) or [CLion](https://www.jetbrains.com/en-us/clion/features/cmake-support.html) have support for CMake built-in.
+  There is even a [standalone GUI](https://cmake.org/runningcmake/) that you can try.
 - Improved caching:
   Unnecessary recompilation is annoying and a waste of time.
   To avoid this, we had a custom caching solution in SCons, which cached the compilation units based on the git branch you're working on.
   This solution took a lots of disk space and also had occasional issues with failed builds.
   With CMake, we added support for [ccache](https://ccache.dev/) (and [clcache](https://github.com/frerich/clcache) on Windows), which makes development significantly faster.
 - Arguably, CMake is vastly more popular than SCons, so we hope that more people be able to build and contribute to Mixxx.
 
-Most of our core developers already switched over to CMake as their daily driver when working on Mixxx, but since it hasn't been as widely tested as we'd like, we label the CMake support as "experimental" for the 2.3 and stick with SCons for the release builds.
+Note that there are still some issues with building Mixxx on OSX, so if you own a Mac we'd appreciate if you help us out.
```

Comment:
```suggestion
Note that there are still some issues with building Mixxx on macOS with CMake, so if you own a Mac we'd appreciate if you help us out.
```
> Apple shortened the name to "OS X" in 2012 and then changed it to "macOS" in 2016, adopting the nomenclature that they were using for their other operating systems, iOS, watchOS, and tvOS.

https://en.wikipedia.org/wiki/MacOS